docs/predictive-analytics/model-retraining/index.md

```diff
@@ -0,0 +1,58 @@
+---
+seo:
+  title: Machine Learning Model Retraining
+  description: This recipe demonstrates how to use ksqlDB to evaluate the predictions of a machine learning model and send data to retrain the model when needed.
+---
+
+# Machine Learning Model Retraining
+
+Machine learning provides valuable insights to an organization and tools like Apache Kafka, Kafka Connect, and kslqDB allow us to build powerful machine learning pipelines. We can also use these tools to extend an existing machine learning pipeline. In this recipe we'll use Connect and ksqlDB to read the results of an existing pipeline, determine the accuracy of said results, and send data to retrain our model.
```

Comment:
@daveklein can you please add a link to the blog post for credit to the author and further reading?

Reply:
Done.